terraform/app/modules/paas/main.tf

```diff
@@ -11,6 +11,7 @@ resource cloudfoundry_app web_app {
   docker_image = var.app_docker_image
   health_check_type = "http"
   health_check_http_endpoint = "/check"
+  health_check_timeout = 60
```

Comment:
This is from TVS, to stop PaaS from killing our app: https://github.com/DFE-Digital/teaching-vacancies/pull/2927

Reply:
Is this 60s? What is the default? 1s? Do we need 60 times the boost?

Reply:
This is 60s, yes. There's a different between startup and running. Current running is 1s. Looking at the TVS PR, I believe this is set to 60s because it sets both startup and running, and we need more than e.g. 10s for startup